drivers/staging/vc04_services/bcm2835-isp/Kconfig

```diff
@@ -0,0 +1,11 @@
+config VIDEO_ISP_BCM2835
+	tristate "BCM2835 ISP support"
+	depends on MEDIA_SUPPORT
+	depends on VIDEO_V4L2 && (ARCH_BCM2835 || COMPILE_TEST)
+	select BCM2835_VCHIQ_MMAL
+	select VIDEOBUF2_DMA_CONTIG
+	select V4L2_MEM2MEM_DEV
+	help
+	  Say Y here to enable the V4L2 ISP driver for
+	  Broadcom BCM2835 SoC. This operates over the VCHIQ interface
+	  to a service running on VideoCore.
```

Comment:
Normally includes the resulting module name as well.

Reply:
Could you point to a template or example for this?  I cannot seem to find any to follow?

Reply:
It seems to vary whether they do or not., so obviously not too critical
https://github.com/raspberrypi/linux/blob/rpi-5.4.y/drivers/media/platform/Kconfig#L109 is an example, as are numerous of the other drivers in that Kconfig.